deNBI/cloud-portal-webapp
# saving

@dweinholz 
To test:
in new instance choose rstudio as template, start vm. in how to connect modal a string should be shown in resenv tab. in /admin/ you should see a backend object with all information. When deleting the vm, the backend will be deleted in forc and in /admin/ vm field should be null.

Needs:
https://github.com/deNBI/cloud-api/pull/953
https://github.com/deNBI/cloud-portal-client/pull/181

Try to fulfill the following points before the Pull Request is merged:

- [ ] Give a meaningfull description for the PR
- [ ] The PR is reviewed by one of the team members.
- [ ] It must be checked if anything in the Readme must be adjusted (development-, production-, setup).
- [ ] It must be checked if any section in the wiki (https://cloud.denbi.de/wiki/) should be adjusted.
- [ ] If the PR is merged in the master then a release should be be made.
- [ ] The PR is responsive on smaller screens.
- [ ] If the requirements.txt have changed, check if the patches still work
- [ ] If the new code is well commented
- [ ] In case the code is not well commented: An respectice commenting issue with tag "important" is opened.
- [ ] If a squash of commits is required, it has been performed or will be performed at final merge
- [ ] Finally a second team member checks if all requirements met


For releases only:

- [ ] If the review of this PR is approved and the PR is followed by a release then the .env file 
  in the cloud-portal repo should also be updated.
